assets/src/edit-story/components/keyboardShortcutsMenu/constants.js

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { KeyboardShortcut as KeyboardShortcutIcon } from '../../icons';
+
+export const KEY_SIZE = {
+  NORMAL: 24,
+  LARGE: 56,
+};
+
+const SPECIAL_KEYS = {
+  COMMAND: (
+    <KeyboardShortcutIcon
+      aria-label={__('Command key', 'web-stories')}
+      width="10"
+      height="10"
+    />
+  ),
+  ENTER: 'Enter',
+  SHIFT: 'Shift',
+  DELETE: 'Delete',
+};
+
+export const KEYBOARD_SHORTCUTS = {
+  headers: [
+    {
+      label: __('Keyboard Shortcuts', 'web-stories'),
+      shortcut: [SPECIAL_KEYS.COMMAND, '/'],
+    },
+  ],
+  panels: [
+    {
+      // This shortcut panel should remain disabled until
+      // this ticket has been completed:
+      // https://github.com/google/web-stories-wp/issues/4241
+      disabled: true,
```

Comment:
Using `disabled` we can hide shortcuts that aren't yet implemented and pop them in real fast once the work is complete.